### ISSUE TITLE
[BUG]The loader simply does nothing

### ISSUE THREAD

**Issue by LasagnaIsYes** [MSCLoaderPro/MSCModLoaderPro#29]
I downloaded the installer and installed the loader, and the files are in the game folder just nothing is happening, I launch MSC and there's no "Mods" button and there's no "now loading mods 1995" or smth like that.

How to do:
1. Download the installer
2. Install
3. Launch the game

Absolutely nothing changes from the main game.

![Screenshot_1](https://user-images.githubusercontent.com/88395875/128054733-a7646eeb-8ab4-47d1-86b1-2fe76e4d2d0e.png)
![Screenshot_2](https://user-images.githubusercontent.com/88395875/128054805-94d62719-a0dd-40c4-8a96-5f6787133306.png)

I do not have any "output_log.txt" in the game folder.

It could be some other problem so please just tell me any solutions and I'll try them out, I've seen other people having the same problem.

**Comment by headcase0570** [comment]
Not sure if the author of MSCLoaderPro will appreciate me saying this, but at the moment the loader seems broken for certain people. I've opened an issue in early june that's yet to be resolved. Maybe you'll have better luck with the regular MSCLoader.

**Comment by LasagnaIsYes** [comment]
> Not sure if the author of MSCLoaderPro will appreciate me saying this, but at the moment the loader seems broken for certain people. I've opened an issue in early june that's yet to be resolved. Maybe you'll have better luck with the regular MSCLoader.

Yeah I've tried the old mod loader and it is compatible for most mods, but some really good mods only work on the new mod loader like Tangerine FZ-120 Pickup.

**Comment by LasagnaIsYes** [comment]
Another addition to the post, if you are having the same problem, use older files of the same mods, for nexus mods go to files, scroll down to old files and get the newest release before the release of "Mod Loader Pro" (April 30th, 2021), then it should work fine.

**Comment by Tresquel** [comment]
You can try the manual install or just trying to uninstall it and installing it again

**Comment by LasagnaIsYes** [comment]
> You can try the manual install or just trying to uninstall it and installing it again

Tried many times, still doesn't work.

**Comment by Tresquel** [comment]
> Tried many times, still doesn't work.

Well then I have no clue what's the case.

**Comment by Fredrik95** [comment]
According to that log file, you're still using the old mod loader.. Did you uninstall that properly before installing the new one?

Try completely removing the old loader, then verify your game through Steam (backup any files as necessary), then reinstall the new mod loader.
Be sure to run the game as an administrator as well.

Also, I apologize for the late reply!
